modules/gltf/editor/editor_scene_importer_blend.cpp

```diff
@@ -301,10 +301,9 @@ Node *EditorSceneFormatImporterBlend::import_scene(const String &p_path, uint32_
 
 #ifndef DISABLE_DEPRECATED
 	bool trimming = p_options.has("animation/trimming") ? (bool)p_options["animation/trimming"] : false;
-	bool remove_immutable = p_options.has("animation/remove_immutable_tracks") ? (bool)p_options["animation/remove_immutable_tracks"] : true;
-	return gltf->generate_scene(state, (float)p_options["animation/fps"], trimming, remove_immutable);
+	return gltf->generate_scene(state, (float)p_options["animation/fps"], trimming, false);
 #else
-	return gltf->generate_scene(state, (float)p_options["animation/fps"], (bool)p_options["animation/trimming"], (bool)p_options["animation/remove_immutable_tracks"]);
+	return gltf->generate_scene(state, (float)p_options["animation/fps"], (bool)p_options["animation/trimming"], false);
```

Comment:
I suspect that the original implementation of remove_immutable_tracks is completely ignored due to enforcing `false` here and causing the issue in #91893. I wonder if it needs to be enabled depending on the options as `(bool)p_options["animation/remove_immutable_tracks"]);`, in this case such as if the external pose is not being used as a reference for rest. Though? @lyuma @fire

Reply:
That argument enables the old option which applies before silhouette fixer and hence will remove tracks inappropriately before silhouette fixer changes the pose. When doing retargeting, it is important for this to be false, and it can always be false.

The remove immutable tracks code in resource_importer_scene has code that should remove scale tracks if they match the rest pose.

Reply:
But has this PR also changed the order of ApplyNodeScale and RemoveImmutableTrack when you changed the order?